### ISSUE TITLE
Add ability to process formatting.

### ISSUE THREAD

**Issue by shacknetisp** [IceflowRE/irc-discord-relay#2]
Currently the discord input and output is not formatted, so it appears in raw form and is not translated.
Adding to ability to @ users and display @'s correctly would be very useful.
An example is included:

IRC log:
```
21:54 <rediscord> <Beha> <@!331517669662654464> nice
21:54 <@behalebabo> uhoh
21:55 <rediscord> <Beha> <@396094340176543755>
21:55 <rediscord> <Beha> <:pzap:396123516875046912>
21:55 <@behalebabo> @Beha
```

The first two @'s were trying to @ a person in the server. They appeared in IRC like this.
The line with 'pzap' was attempting to use an inline image.
Note that the final `@Beha` did not notify the user in discord.
--- Edit: @IceflowRE
https://github.com/IceflowRE/irc-discord-relay/tree/discordMention

**Comment by IceflowRE** [comment]
Fixed except the Discord mentioning from IRC.

**Comment by IceflowRE** [comment]
Finaly fixed by: bca132d72ba5e068a9710a7e34520e9675ea4fc7
